flask/boxwise_flask/models/log.py

```diff
@@ -0,0 +1,12 @@
+from boxwise_flask.db import db
+from peewee import CharField, DateTimeField, TextField
+
+
+class Log(db.Model):
+    content = TextField()
+    ip = CharField(null=True)
+    date = DateTimeField(column_name="logdate", null=True)
+    user = CharField(null=True)
```

Comment:
@aerinsol @HaGuesto what user is meant here since it's not the User table that is referenced?

Reply:
@pylipp I think this field used as reference note of the user name in the log, since the user may be deleted or changed later on the system, but @HaGuesto can give more insight on that.